src/components/MapboxMap/MapboxMap.vue

```diff
@@ -118,7 +142,18 @@ function initMap(accessToken: string) {
           promoteId: "spaceId",
         },
         layout: {
-          "icon-image": "marker-icon",
+          "icon-image": [
+            "match",
+            ["get", "buildingOccupancy"],
+            "quiet",
+            "map-marker-quiet",
+            "busy",
+            "map-marker-busy",
+            "crowded",
+            "map-marker-crowded",
+            "map-marker", //default
```

Comment:
should we rename this one to `"map-marker-default"` so the variable and file name are self-explanatory?